swift/ql/src/queries/Security/CWE-328/WeakPasswordHashing.qhelp

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE qhelp PUBLIC
+        "-//Semmle//qhelp//EN"
+        "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>
+            Using a insufficiently computationally hard hash function can leave data
+            vulnerable, and should not be used for password hashing.
+        </p>
```

Comment:
```suggestion
        <p>
            Hash functions that are not sufficiently computationally hard can leave data vulnerable. You should not use such functions for password hashing.
        </p>
```

Minor: breaks sentence up slightly, avoids stacked nouns, makes slightly more active

Reply:
Just a minor note as well: in this sentence you mention the idea of a function being "sufficiently computationally hard" whereas the rest of the article references the concept of computational hardness as more of a binary is/isn't. I'm not sure if it makes sense to change this to be more like the rest of the article in that regard. In any case don't feel the need to respond to this note if it's not useful.

Reply:
I have attempted to address the inconsistencies around phrasing of "computationally hard", by adding the world "sufficiently" in a few more places.  Please don't hesitate to make further suggestions on the resulting text.